tariff/ngeso/api.go

```diff
@@ -122,9 +123,9 @@ func (r NationalIntensityResult) Results() []CarbonIntensityForecastEntry {
 }
 
 type CarbonIntensityForecastEntry struct {
-	ValidityStart shortRFC3339Timestamp `json:"from"`
-	ValidityEnd   shortRFC3339Timestamp `json:"to"`
-	Intensity     CarbonIntensity       `json:"intensity"`
+	ValidityStart shortrfc3339.ShortRFC3339Timestamp `json:"from"`
```

Comment:
Dunno what else the RFC covers, but wouldn't

    rfc3339.Short

be nice?

Reply:
Maybe? Feels like we'd just be confusing the very niche use case of the package with the already existing rfc3339 functionality in standard `time`